pkgs/applications/editors/kile/default.nix

```diff
@@ -42,7 +42,6 @@ stdenv.mkDerivation rec {
   meta = {
     description = "User-friendly TeX/LaTeX authoring tool for the KDE desktop environment";
     homepage = "https://www.kde.org/applications/office/kile/";
-    maintainers = with lib.maintainers; [ fridh ];
```

Comment:
Please keep the maintainers line, sot hat it is obvious that the package has no maintainer.

Reply:
The linter does not like that and fails the PR.

Reply:
Why?

Reply:
Can we stop with the nitpicking already? It's exhausting.

Reply:
This was not a blocker from my side, I was legitimately asking why the linter complained about this.

Reply:
@AndersonTorres I am not aware that we are checking this on nixpkgs level, likely some editor linter but I personally think it is fine to ignore that to explicitly document that the package have no maintainer.